pkg/kubectl/cmd/auth/reconcile.go

```diff
@@ -214,6 +225,18 @@ func (o *ReconcileOptions) RunReconcile() error {
 
 		return nil
 	})
+}
 
-	return err
+func (o *ReconcileOptions) setDefaultPrinting() {
```

Comment:
Why isn't this done in `Complete`?  Embedders can set it as they see fit.

Reply:
Because `Complete` is CLI / cobra / "use a file as input" specific.  I would expect embedders to instantiate the struct directly followed by `Validate` and `RunReconcile`.  This just turns the printing into no-ops so that `RunReconcile` does not panic.

Reply:
> Because Complete is CLI / cobra / "use a file as input" specific. I would expect embedders to instantiate the struct directly followed by Validate and RunReconcile. This just turns the printing into no-ops so that RunReconcile does not panic.

We decided not to do that a couple years back.  You embed, you need to know what you're doing.  `Run*` should not mutate its options.

Reply:
Ah, OK.  Then I can just remove this method since I do not see how to add it to `Complete` in a way that would be useful.